index.js

```diff
@@ -1,5 +1,7 @@
 function locate(array, target) {
-  // do work here
+  let flattenArray = [].concat.apply([], array);
+  let flattenMore = [].concat.apply([], flattenArray);
```

Comment:
This only works for so many levels deep, can you rewrite this to handle any depth?